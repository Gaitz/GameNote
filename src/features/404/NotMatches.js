import React from "react"

export default function NotMatches() {
  return (
    <article>
      <h1>404</h1>
    </article>
  )
}
