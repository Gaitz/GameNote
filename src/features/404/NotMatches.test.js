import React from "react"
import renderer from "react-test-renderer"
import NotMatches from "./NotMatches"

test("404 component snapshot testing", () => {
  const component = renderer.create(<NotMatches></NotMatches>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
