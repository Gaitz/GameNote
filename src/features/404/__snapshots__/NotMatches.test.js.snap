// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 component snapshot testing 1`] = `
<article>
  <h1>
    404
  </h1>
</article>
`;
